tests/metrics/de/test_reference_based_text_quality.py

```diff
@@ -0,0 +1,155 @@
+import os
+from unittest.mock import Mock, patch
+
+import pytest
+from openai.types import CreateEmbeddingResponse
+
+from langcheck.metrics.de import rouge1, rouge2, rougeL, semantic_similarity
+from tests.utils import is_close
+
+################################################################################
+# Tests
+################################################################################
+
+
+@pytest.mark.parametrize(
+    "generated_outputs,reference_outputs",
+    [
+        ("Die Katze saß auf der Matte.", "Die Katze saß auf der Matte."),
+        (["Die Katze saß auf der Matte."], ["Die Katze saß auf der Matte."]),
+    ],
+)
+def test_semantic_similarity_identical(generated_outputs, reference_outputs):
+    metric_value = semantic_similarity(generated_outputs,
+                                       reference_outputs,
+                                       model_type="local")
+    assert 0.99 <= metric_value <= 1
+
+
+@pytest.mark.parametrize(
+    "generated_outputs,reference_outputs",
+    [
+        ("Die KATZE saß auf der MATTE.", "Die Katze saß auf der Matte."),
+        (["Die KATZE saß auf der MATTE."], ["Die Katze saß auf der Matte."]),
+    ],
+)
+def test_semantic_similarity_case_sensitivity(generated_outputs,
+                                              reference_outputs):
+    # nb: the German model is case-sensitive!
+    metric_value = semantic_similarity(generated_outputs,
+                                       reference_outputs,
+                                       model_type="local")
+    assert 0.6 <= metric_value <= 0.7
+
+
+# fixme: why is this so high in similarity???
```

Comment:
is this still true? from the assertion below it looks like the similarity is pretty low now?

Reply:
good point, removed